alarm.graphqls

```diff
@@ -24,14 +24,20 @@ type AlarmMessage {
     scope: Scope
     id: ID!
     message: String!
+    tags: [KeyValue!]!
 }
 
 type Alarms {
     msgs: [AlarmMessage!]!
     total: Int!
 }
 
+input AlarmTag {
+    key: String!
+    value: String
+}
+
 extend type Query {
     getAlarmTrend(duration: Duration!): AlarmTrend!
-    getAlarm(duration: Duration!, scope: Scope, keyword: String, paging: Pagination!): Alarms
+    getAlarm(duration: Duration!, scope: Scope, keyword: String, paging: Pagination!, tags: [AlarmTag]): Alarms
```

Comment:
If you are asking for this change, please submit another new issue, we need to notify UI side PMC members, and get their approval.

Reply:
The old tag discussion is only related to notification, it makes sense to add this on about native/default UI. But still, the process is required.

Reply:
I will open a issue later.

Reply:
@wu-sheng  Related issue https://github.com/apache/skywalking/issues/6821